### DISCUSSION TITLE
Plugin: Pi-hole

### DISCUSSION THREAD

**Discussion by coolapso** [StreamController/StreamController#35] [Ideas]
Would be great to have a pihole plugin that could: 
* Enable / disable pihole
* Show some of the available metrics
* Show errors

**Comment by Core447** [comment]
Added basic Pi-hole plugin that can enable/disable pihole and show metrics: https://github.com/StreamController/Pi-Hole

Which errors would you like to show the plugin?

**Reply by coolapso** [reply]
I'd say just a generic counter to let know if there's any warning/notification (not even sure if that is exposed  on the pihole api,  so might not even be possible).

**Reply by Core447** [reply]
I don't think that's exposed, sry

**Reply by coolapso** [reply]
Yeah i'd figure that probably not. well just the toggle / info / disable works wonders! thank you! :D

**Reply by Core447** [reply]
You're welcome
